recipe/meta.yaml

```
@@ -40,9 +39,11 @@ requirements:
     - python
     - pip
     - wheel
-    - numpy {{ numpy }}
-    - cython 0.29
+    - numpy 2.0 # [py<313]
+    - numpy 2.1 # [py==313]
+    - cython >=3.0.11
     - zlib {{ zlib }}
+    - zlib-ng 2.0
     - bzip2 {{ bzip2 }}
     - xz {{ xz }}
     - zstd 1.5.5
```

```suggestion
    - zstd 1.5.6
```
https://github.com/cgohlke/imagecodecs/tree/v2024.9.22?tab=readme-ov-file#requirements

---

or maybe pin to cbc.yaml if that will work

---

I'll try with `zstd {{ zstd }}`